### PR TITLE
Several spelling suggestions

### PR DESCRIPTION
**Pull request by mdeweerd**
I collected about 10 spelling suggestions.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -55750,6 +55761,7 @@ thogh->though, thigh,
 thoght->thought
 thoghts->thoughts
 thonic->chthonic
+thoose->those
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:55764]:
```suggestion
thoose->those, choose,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -55831,6 +55843,7 @@ thresolds->thresholds
 thresshold->threshold
 thressholds->thresholds
 thrid->third
+thridparty->thirdparty
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:55846]: Not really a word, it's usually `third party` or `third-party`.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -54877,6 +54887,7 @@ tagggen->taggen
 tagnet->tangent
 tagnetial->tangential
 tagnets->tangents
+tagret->target
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:54890]:
```suggestion
tagret->target
tagrets->targets
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -26551,6 +26553,7 @@ footprinst->footprints
 foound->found
 foppy->floppy
 foppys->floppies
+foraign->foreign
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:26556]:
```suggestion
foraign->foreign
foraigner->foreigner
foraigners->foreigners
```

**Review by DimitriPapadopoulos — COMMENTED** [review]
What would you think of adding `simulare->simulate` and `dissimulare->dissimulate`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt:55764]: IMHO the c an t are quite far from each other and therefore thoose->choose is a bit far-fetched.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt:55846]: I'll update that - I've seen that as a single word a lot of the time but that's probably 'european english'

**Comment by mdeweerd** [comment]
I still can't accept the suggestions in the UI, I left a comment from the documentation page ( https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request ).

Also added coventional and derivatives thereof.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt]: The [OED](https://www.oed.com/search/dictionary/?scope=Entries&q=third-party) has only `third party`, and [SCOWL (And Friends)](http://wordlist.aspell.net/) does not appear to be including `third-party` either. Other dictionaries such as [Cambridge](https://dictionary.cambridge.org/fr/dictionnaire/anglais/third-party) have both  `third party` and  `third-party`.

I really don't know. Would you like to suggest `thirdparty-> third party` only, or `thirdparty-> third party, third-party,`?

Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt]: I've added those suggestions.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt]: Not on Dvorak keyboards though:

The modern Dvorak layout (U.S. layout)

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt]: Ah, you have already chosen the latter. Great!

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt]: Surprising keyboard, thanks for the info.  It was added anyway.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @mdeweerd.

You probably cannot apply suggestions from the UI because of a GitHub bug.

**Pull request merged**